reana_server/rest/workflows.py

```diff
@@ -2189,15 +2186,9 @@ def get_workflow_disk_usage(workflow_id_or_name):  # noqa
         workflow = _get_workflow_with_uuid_or_name(workflow_id_or_name, str(user.id_))
         summarize = bool(parameters.get("summarize", False))
         block_size = parameters.get("block_size")
-        reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
-        if reana_fs.exists(workflow.workspace_path):
-            absolute_workspace_path = reana_fs.getospath(workflow.workspace_path)
-            disk_usage_info = get_workspace_disk_usage(
-                absolute_workspace_path, summarize=summarize, block_size=block_size
-            )
-        else:
-            raise ValueError("Workspace does not exist.")
-
+        disk_usage_info = workflow.get_workspace_disk_usage(
```

Comment:
Something weird happens with this:
```console
$ reana-client du -w helloworld-cwl
SIZE   NAME
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
8.0K   .   
0      .   
0      .   
12K    .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
4.0K   .   
16K    .   
16K    .   
4.0K   . 
```
Seems like names are not being parsed correctly?